supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -173,9 +173,42 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
 
   import mouse.all._
 
-  private def generateGcsLocalizationScript(inputs: List[PipelinesApiInput])(implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
-    val bundleFunction = (gcsLocalizationTransferBundle(gcsTransferConfiguration) _).tupled
-    generateGcsTransferScript(inputs, bundleFunction) |> bracketTransfersWithMessages("Localization")
+  private def generateGcsLocalizationScript(inputs: List[PipelinesApiInput], referenceFilesMapping: PipelinesApiReferenceFilesMapping)(implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
+    val referenceInputsToMountedPaths = PipelinesApiReferenceFilesMapping.getReferenceInputsToMountedPathMappings(referenceFilesMapping, inputs)
+    val referenceFilesLocalizationScript = {
+      val symlinkCreationCommands = referenceInputsToMountedPaths map {
+        case (input, absolutePathOnRefDisk) =>
+          s"mkdir -p ${input.containerPath.parent.pathAsString} && ln -s $absolutePathOnRefDisk ${input.containerPath.pathAsString}"
```

Comment:
We also need to validate the crc32c at this point to make sure what we're about to reference on the references disk matches what we would have downloaded from GCS. Otherwise, we have to fall back to full localization.

Reply:
Maybe we should introduce disk image versioning (name them like broad-references-disk-image-v1) instead?